src/compas/files/base_reader.py

```diff
@@ -10,6 +10,8 @@
 except ImportError:
     from urllib import urlretrieve
 
+import binaryornot.check
```

Comment:
i think this is a good idea, but we should check when it identifies a file as binary.
from reading through the code quickly, it seems that if more than 5 % of the code is non-ascii it is considered binary. while probably sufficient in real-world examples, ply headers (ascii) can be quite long even if the described object is very simple. not sure if this then would pass the test. but perhaps we should cross that bridge when we get there... :)

Reply:
Yeah, I think that we can have general checks in BaseReader while some more specific should be in the file reader. 

A well formatted binary PLY should have `format binary_little_endian 1.0` or 
`format binary_big_endian 1.0` in the next line. :)

Reply:
> i think this is a good idea [...]

It was your idea ;).